public/src/components/channelManagement/bannerDesigns/utils/defaults.ts

```diff
@@ -1,7 +1,9 @@
 import { BannerDesign } from '../../../../models/bannerDesign';
 import { stringToHexColour } from '../../../../utils/bannerDesigns';
 
-export const DEFAULT_BANNER_DESIGN = {
+export const createDefaultBannerDesign = (name: string): BannerDesign => ({
```

Comment:
Ah great that now the colour fields in the form are `HexColour`s this can all be simplified/duplication removed 👍